src/Symfony/Bundle/TwigBundle/ContainerAwareRuntimeLoader.php

```diff
@@ -11,8 +11,8 @@
 
 namespace Symfony\Bundle\TwigBundle;
 
+use Psr\Container\ContainerInterface;
 use Psr\Log\LoggerInterface;
```

Comment:
@stof This means deprecating this class and bump the twig requirement to 2.x in TwigBundle to use the twig class instead, right?